### ISSUE TITLE
PouchDB <-> CouchDB CORS in Chrome 65

### ISSUE THREAD

**Issue by t-knapp** [pouchdb/pouchdb#7163]
### Issue
Chrome 65 skrews up CORS preflight to CouchDB.
Sync to CouchDB does not work then.
Do anybody face this problem also?

### Info
- Environment: browser
- Platform: Chrome 65
- Adapter: IndexedDB
- Server: CouchDB
- PouchDB: 6.1.0

### Reproduce
Try to sync() to a remote CouchDB with Chrome 65 installed.

**Comment by t-knapp** [comment]
Fun fact: Did not change anything to CORS settings of CouchDB (v2.0.1). The set up works seamlessly with Chrome 64. Will update CouchDB to 2.1.1 and also try add-cors-to-couchdb.

**Comment by t-knapp** [comment]
Tested with PouchDB 6.4.3, CouchDB 2.1.1 and add-cors-to-couchdb. CORS problem is still there. 

`Failed to load http://127.0.0.1:5984/mydb/: Response to preflight request doesn't pass access control check: The 'Access-Control-Allow-Origin' header has a value 'null' that is not equal to the supplied origin. Origin 'null' is therefore not allowed access.`

Works with Chrome 64. @daleharvey I know, it's not a PouchDB issue, I created this issue here to ask whether other people face this problem.
